lib/tests/test.js

```diff
@@ -74,7 +76,24 @@ class Test {
     this.engine.contractsManager.build(() => {
       self.builtContracts = cloneDeep(self.engine.contractsManager.contracts);
       self.compiledContracts = cloneDeep(self.engine.contractsManager.compiledContracts);
-      callback();
+      self.buildImportFile(callback);
+    });
+  }
+
+  buildImportFile(callback) {
+    let content = 'module.exports = {';
+    Object.keys(this.builtContracts).forEach(className => {
+      content += `\n\t"${className}": embark.require('Embark/contract/${className}'),`;
```

Comment:
To resolve the issue that this would create an Embark folder in the dapp, couldn’t we change 
```
content += `\n\t"${className}": embark.require('Embark/contract/${className}'),`;
```
to
```
content += `\n\t"${className}": embark.require('.embark/contract/${className}'),`;
```?